lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
+
+var assert = require('assert-plus');
+
+var RestifyClient = require('./restifyclient');
+
+function VOLAPI(options) {
+    RestifyClient.call(this, options);
+    this.url = options.url;
+}
+
+util.inherits(VOLAPI, RestifyClient);
+
+
+VOLAPI.prototype.close = function close() {
+    this.client.close();
+};
+
+VOLAPI.prototype.createVolume =
+    function createVolume(params, options, callback) {
```

Comment:
Yes, that's what I meant. E.g. https://github.com/joyent/sdcadm/blob/f7e0a0eeedb8666032a3151caa9a2a31736ae3fe/lib/common.js#L40-L75